public/caesar-shift/encrypt.py

```diff
@@ -0,0 +1,53 @@
+import string
+from encrypt_library import *
+#START#
```

Comment:
`#START#` and `#END` are how I parse only a portion of the file into the module for the users to see. There are tests and imports which I do not find necessary for students to worry about, so I hide them through specifying a specific range of the file to look at. You can see this in action in the `convert_to_html` function in `password.js`.